discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/NoticeQueryResponse.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * NoticeQueryResponse.
+ */
+public class NoticeQueryResponse extends GenericModel {
+
+  @SerializedName("matching_results")
+  private Long matchingResults;
+  private List<NoticeQueryResult> results;
+  private List<QueryAggregation> aggregations;
```

Comment:
assuming that a `NoticeQueryRequest` takes all the same parameters as `DocumentQueryRequest`, this should also have `List<Passage> passages;` - i realize this  is not in the swagger